docs/INSTALL.md

```diff
@@ -4,7 +4,8 @@
   - [Installing a binary release](#installing-a-binary-release)
   - [Building a tagged version with Docker](#building-a-tagged-version-with-docker)
   - [Building a development version from source](#building-a-development-version-from-source)
-    - [Preliminaries for installing from source](#preliminaries-for-installing-from-source)
+    - [Installing Go](#installing-go)
+    - [Go modules](#go.modules)
```

Comment:
The link probably needs to be `#go-modules`.

Reply:
Fixed